pkgs/tools/bluetooth/blueberry/default.nix

```diff
@@ -0,0 +1,83 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, bluez-tools
+, cinnamon
+, gnome3
+, gobject-introspection
+, intltool
+, pavucontrol
+, python3Packages
+, rfkill
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "blueberry";
+  version = "1.3.9";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "0llvz1h2dmvhvwkkvl0q4ggi1nmdbllw34ppnravs5lybqkicyw9";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    python3Packages.wrapPython
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    bluez-tools
+    cinnamon.xapps
+    gnome3.gnome-bluetooth
+    python3Packages.python
+    rfkill
+  ];
+
+  pythonPath = with python3Packages; [
+    dbus-python
+    pygobject3
+    setproctitle
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out
+    cp -a etc usr/* $out
+
+    find $out -type f -exec sed \
+      -e "s;/usr/bin/pavucontrol;${pavucontrol}/bin/pavucontrol;" \
+      -e "s;/usr/bin/rfkill;${rfkill}/bin/rfkill;" \
+      -e "s;/usr/lib/blueberry;$out/lib/blueberry;" \
+      -e "s;/usr/sbin/rfkill;${rfkill}/bin/rfkill;" \
```

Comment:
Should this replace `bt-adapter` with `${bluez-tools}/bin/bt-adapter` too?

Reply:
Done.